Register prefix `expltools`

Witiko opened the pull request:
This PR registers the prefix ~~`expl3tools`~~ `expltools`.

muzimuzhi commented:
It seems currently `expl3tools` is not yet used as prefix in https://github.com/witiko/expl3tools/. Are you really going to change catcode of `3` locally and define things like `\l__expl3tools_..._tl` and `\expl3tools_...:n`?

Witiko commented:
@muzimuzhi That's the plan.

Witiko commented on l3kernel/doc/l3prefixes.csv:
```diff
@@ -78,6 +78,7 @@ etex,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https:
 etl,etl,Jonathan P. Spratte,https://github.com/Skillmon/ltx_etl,git@github.com:Skillmon/ltx_etl.git,https://github.com/Skillmon/ltx_etl/issues,2021-08-16,2021-08-16,
 exp,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https://github.com/latex3/latex3.git,https://github.com/latex3/latex3/issues,2012-09-27,2012-09-27,
 expl,l3kernel,The LaTeX Project,https://www.latex-project.org/latex3.html,https://github.com/latex3/latex3.git,https://github.com/latex3/latex3/issues,2012-09-27,2012-09-27,
+expl3tools,expl3tools,Vít Starý Novotný,https://github.com/witiko/expl3tools/,https://github.com/witiko/expl3tools.git,https://github.com/witiko/expl3tools/issues,2024-06-26,2024-06-26,Development tools for expl3 programmers
```
```suggestion
expltools,expltools,Vít Starý Novotný,https://github.com/witiko/expltools/,https://github.com/witiko/expltools.git,https://github.com/witiko/expltools/issues,2024-06-26,2024-06-26,Development tools for expl3 programmers
```

Witiko commented:
If you'd like to keep the prefix reserved for internal use, please let me know and I will come up with a different one.

> It seems currently `expl3tools` is not yet used as prefix in https://github.com/witiko/expl3tools/.

I am registering early, so that we can quibble about the prefix and potentially change it without a large impact on the code base.

Witiko commented:
> It seems currently `expl3tools` is not yet used as prefix in https://github.com/witiko/expl3tools/. Are you really going to change catcode of `3` locally and define things like `\l__expl3tools_..._tl` and `\expl3tools_...:n`?

@muzimuzhi: In commit d8d6536 from this PR, I changed the prefix from `expl3tools` to just `expltools`.
In the merged PR https://github.com/Witiko/expltools/pull/3, I changed the code, so that it uses the prefix `expltools`.

@josephwright: Are there any remaining concerns here?

car222222 commented:
No, nothing more to say.  Go for it! 
(But `explatypus` would be good too:-).

Witiko commented:
> (But `explatypus` would be good too:-).

Explatypus may still make a comeback as one tool underneath the expltools umbrella. :wink:

car222222 commented:
🙃
